server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/model/MailboxFactoryTest.java

```diff
@@ -197,6 +198,35 @@ public void getNamespaceShouldReturnPersonalNamespaceWhenUserMailboxPathAndUserM
             .isEqualTo(MailboxNamespace.personal());
     }
 
+    @Test
+    public void buildShouldRelyOnPreloadedMailboxes() throws Exception {
+        MailboxPath inbox = MailboxPath.inbox(user);
+        Optional<MailboxId> inboxId = mailboxManager.createMailbox(inbox, mailboxSession);
+        Optional<MailboxId> otherId = mailboxManager.createMailbox(MailboxPath.forUser(user, "INBOX.child"), mailboxSession);
```

Comment:
what about add a `child` method to MailboxPath that given a mailbox name would create a child MailboxPath?